airbyte-integrations/connectors/source-sense-data/manifest.yaml

```diff
@@ -0,0 +1,78 @@
+version: 4.6.2
+
+type: DeclarativeSource
+
+description: Conector de leitura dos dados Sense Data
```

Comment:
Connector docs are not translated to different languages ;-( probably best to start with a sentence in English, but ok to keep this right after.

Reply:
Ok, added to backlog